python file>.py

```diff
@@ -0,0 +1,57 @@
+fs_pairs = {'bob': 'bobby',
+            'apple': 'tree',
+            'dog': 'wolf'}
+choice = None
+
+while choice != '1':
+    print('''
+
+    Your choices right now:
+
+    1: Exit
+    2: add son father pair
+    3: replace a son's father with another 
+    4: delete son father pair
+    5: view son father pairs
+
+    ''')
+    choice = input('What is your choice? : ')
+    if choice == '1':
+        print('Bye')
+    elif choice == '2':
+        print('Your choice was add son father pair')
+        add = input('What would the son\'s name be? : ')
+        add2 = input('What would ' + add + ' \'s father\'s name be? : ')
```

Comment:
`add` and `add2` are not descriptive variable names. Please provide more descriptive variable names.

This might be a good read: https://erik-engheim.medium.com/the-case-against-descriptive-variable-names-3c09dee0bac5

Reply:
changed it to son_add and father_add

Reply:
I read the article thing 👍